app/renderer/reducers/transactions.ts

```diff
@@ -0,0 +1,15 @@
+import * as Actions from "app/renderer/actions/actionNames"
+import { SaveTransactionsAction } from "app/renderer/actions/actionTypes"
+import { prefilledTransactions, ComputedTransactions } from "app/renderer/prefilledTransactions"
+
+const defaultTransactrions: ComputedTransactions = prefilledTransactions
+
+export const transactionsReducer = (state: ComputedTransactions = defaultTransactrions,
+  action: SaveTransactionsAction): ComputedTransactions => {
+  switch (action.type) {
+    case Actions.SAVE_TRANSACTIONS:
+      return prefilledTransactions
+    default:
+      return state
+  }
+}
```

Comment:
Also, I would leave the `prefilledTransactions` out of the reducer (or at least that was the approach taken with the prefilled wallet). The `defaultTransactions` would be an empty array and when handling the `SAVE_TRANSACTIONS` action we just need to return the transactions in the payload of the action itself.